docarray/documents/mesh/mesh_3d.py

```diff
@@ -103,10 +106,24 @@ class MultiModalDoc(BaseDoc):
 
     """
 
-    url: Optional[Mesh3DUrl] = None
-    tensors: Optional[VerticesAndFaces] = None
-    embedding: Optional[AnyEmbedding] = None
-    bytes_: Optional[bytes] = None
+    url: Optional[Mesh3DUrl] = Field(
+        description='URL to a file containing 3D mesh information. Can be remote (web) URL, or a local file path.',
+        example='https://people.sc.fsu.edu/~jburkardt/data/obj/al.obj',
+        default=None,
+    )
+    tensors: Optional[VerticesAndFaces] = Field(
+        description='A tensor object of 3D mesh of type `VerticesAndFaces`.',
+        example=[[0, 1, 1], [1, 0, 1], [1, 1, 0]],
+        default=None,
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='Store an embedding: a vector representation of the 3D mesh.',
+        default=[[1, 0, 1], [0, 1, 1], [1, 1, 0]],
```

Comment:
the embedding is tipically a 1-D vector, so better show that in example

Reply:
Understood but the docs for this document says the below
```
Vertices are points in a 3D space, represented as a tensor of shape (n_points, 3)
```

Reply:
buut vertices are not the embedding. The embedding is a vector semantically representing an object, in this case a 3D mesh represented by its vertices.